lizmap/modules/lizmap/classes/lizmapServices.class.php

```diff
@@ -339,42 +346,27 @@ public function update($data)
         return $modified;
     }
 
-    /**
-     * save the services.
-     */
-    public function save()
+    public function saveIntoIni($ini, $liveIni)
     {
-        // Get access to the ini file
-        $ini = new jIniFileModifier(jApp::configPath('lizmapConfig.ini.php'));
-        $liveIni = new jIniFileModifier(jApp::configPath('liveconfig.ini.php'));
-
-        $dontSaveSensitiveProps = $this->hideSensitiveProperties();
+		$dontSaveSensitiveProps = $this->hideSensitiveProperties();
         $hiddenProps = array();
         if ($dontSaveSensitiveProps) {
             $hiddenProps = array_combine($this->sensitiveProperties, array_fill(0, count($this->sensitiveProperties), true));
-        }
-
+		}
+		
         foreach ($this->properties as $prop) {
             if ($dontSaveSensitiveProps && isset($hiddenProps[$prop])) {
                 continue;
             }
             if (isset($this->globalConfigProperties[$prop])) {
                 list($key, $section) = $this->globalConfigProperties[$prop];
                 $liveIni->setValue($key, $this->{$prop}, $section);
-            } elseif ($this->{$prop} != '') {
+            } elseif (isset($this->{$prop}) && $this->{$prop} != '') {
```

Comment:
without this `isset`, are there some issues? because it seems to me this is useless

Reply:
There is a 'cacheExpiration' in the properties tab that is not a class property so there is an 'Undefined property' PHP Error.

Reply:
So this is a missing name. we should add it.